runtime/arm64.S

```diff
@@ -633,6 +658,11 @@ FUNCTION(caml_c_call_stack_args)
         ldr     TRAP_PTR, Caml_state(exn_handler)
     /* Switch from C to OCaml */
         SWITCH_C_TO_OCAML
+#if defined(WITH_THREAD_SANITIZER)
+        TSAN_PUSH_RETURN_REGS /* Save returned value. */
+        TSAN_EXIT_FUNCTION
+        TSAN_POP_RETURN_REGS
+#endif
```

Comment:
When you have a TSAN_-only sequence, the `ifdef` is technically not necessary because these macros are defined to be empty when TSan is disabled. The style that use use in the rest of the file is to avoid the conditionals in that case.